### PR TITLE
Add calcs

### PR DESCRIPTION
**Pull request by akikuno**
`calcs` is a command line tool to append the minimap2's CS tag to SAM file.
CS tag is one of the tags in SAM format that represents indels and substitutions.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by akikuno** [comment]
@BiocondaBot 
please add label

**Comment by akikuno** [comment]
@BiocondaBot please add label

File: recipes/calcs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3cb00162204b1e05e3ab4c400c8bd6b7793b24834f18badc5396b119440689b3
+
+build:
+  noarch: generic
```
Comment (simonbray) [inline recipes/calcs/meta.yaml:13]:
```suggestion
  noarch: python
```

File: recipes/calcs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  imports:
+    - calcs
+  commands:
+    - calcs --help
+
+about:
+  home: "https://github.com/akikuno/calcs"
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (simonbray) [inline recipes/calcs/meta.yaml:36]:
```suggestion
  license_file: LICENSE
```

File: recipes/calcs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - calcs --help
+
+about:
+  home: "https://github.com/akikuno/calcs"
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "call CS tag to SAM file"
+  doc_url: 
+  dev_url:
```
Comment (simonbray) [inline recipes/calcs/meta.yaml:39]:
```suggestion

```
either remove or add

File: recipes/calcs/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    - calcs
+  commands:
+    - calcs --help
+
+about:
+  home: "https://github.com/akikuno/calcs"
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "call CS tag to SAM file"
```
Comment (simonbray) [inline recipes/calcs/meta.yaml:37]: Maybe a little more detail here?

**Review by simonbray — CHANGES_REQUESTED** [review]

**Review by simonbray — APPROVED** [review]

**Comment by simonbray** [comment]
@bioconda-bot please merge

**Pull request merged**